datasource/src/main/scala/cromwell/pipeline/database/PipelineDatabaseEngine.scala

```diff
@@ -0,0 +1,39 @@
+package cromwell.pipeline.database
+
+import java.sql.Connection
+
+import com.typesafe.config.{Config, ConfigFactory}
+import slick.basic.DatabaseConfig
+import slick.jdbc.JdbcProfile
+
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+class PipelineDatabaseEngine(config: Config = ConfigFactory.load()) extends AutoCloseable {
+
+  private val slickConfig = DatabaseConfig.forConfig[JdbcProfile](path = "postgres_dc", config)
+
+  val profile = slickConfig.profile
+  val database = slickConfig.db
+
+  import profile.api._
+
+  def updateSchema(): Unit = {
+    withConnection(LiquibaseUtils.updateSchema("liquibase/changelog/changelog-master.xml"))
```

Comment:
I suggest we move migration prescription file path to the config and retrieve it from there, what do you think?

Reply:
It's great suggestion, thank you!
Your proposal was implemented.